build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -121,5 +121,29 @@
     "NugetPackageSearchName": "Microsoft.Azure.Cosmos",
     "MinVersion": "3.6.0",
     "MaxVersionExclusive": "4.0.0"
+  },
+  {
+    "IntegrationName": "Serilog",
+    "SampleProjectName": "LogsInjection.Serilog",
+    "NugetPackageSearchName": "Serilog",
+    "MinVersion": "1.0.0",
+    "MinVersionNetCore": "2.0.0",
+    "MaxVersionExclusive": "3.0.0"
+  },
+  {
+    "IntegrationName": "NLog",
+    "SampleProjectName": "LogsInjection.NLog",
+    "NugetPackageSearchName": "NLog",
+    "MinVersion": "1.0.0",
+    "MinVersionNetCore": "4.5.0",
+    "MaxVersionExclusive": "5.0.0"
+  },
+  {
+    "IntegrationName": "log4net",
+    "SampleProjectName": "LogsInjection.Log4Net",
+    "NugetPackageSearchName": "log4net",
+    "MinVersion": "1.0.0",
+    "MinVersionNetCore": "2.0.6",
+    "MaxVersionExclusive": "3.0.0"
```

Comment:
We should add these sample names to the `multiApiProjects` collection in `CompileSamplesLinux`, so we don't build them multiple times (I'll update that target to automatically handle that soon!)

Reply:
Did a small PR to handle this automatically here https://github.com/DataDog/dd-trace-dotnet/pull/1646

Reply:
It looks like this is handled. If there's anything else for me to do, let me know, or you can resolve this